src/components/svelte-react.ts

```diff
@@ -45,6 +44,16 @@ export type ReactProps<Props, Events> = Props & {
   [P in IntrinsicProps]?: JSX.IntrinsicElements['div'][P]
 }
 
+type EventPropsNames = keyof DOMAttributes<unknown>
```

Comment:
I was able to get a list of the props from react here, @fallaciousreasoning using your prototype this actually works so standard react event prop names like `onKeyDown` are valid.